src/components/EditorHeader/ControlPanel.jsx

```diff
@@ -933,6 +934,21 @@ export default function ControlPanel({
               }));
             },
           },
+          {
+            DjangoModels: () => {
```

Comment:
Please replace DjangoModels with "Django Models" so that there's a space in the UI